app/models/game.rb

```diff
@@ -4,4 +4,179 @@ class Game < ApplicationRecord
   belongs_to :player_1, class_name: 'Player', optional: true
   belongs_to :player_2, class_name: 'Player', optional: true
   belongs_to :winning_player, class_name: 'Player', optional: true
+  # Is called after  Base.save on new objects that have not been saved yet (no record exists).
+  after_create :add_white_starting_pieces!
+  after_create :add_black_starting_pieces!
+  # How to fix messed up migrations http://stackoverflow.com/questions/20082002/migrations-are-pending-run-bin-rake-dbmigrate-rails-env-development-to-resol
+  # Populate a game with all the pieces in the correct locations (x_coordinate, y_coordinate)
+  def add_white_starting_pieces!
+    # ----------ADD STARTING WHITE PIECES---------------
+    (1..8).each do |i|
+      pieces << Piece.create(
+        game: self,
+        player: player_1,
+        color: 'white',
+        x_coordinate: i,
+        y_coordinate: 2,
+        type: 'p'
```

Comment:
Type will need to match the Model names used. `Pawn`, `King`, `Queen`, etc.

Rails will use the Type column in the DB to match up with the correct model/class in our models folder.